src/test/exit-codes/compute-phase-errors.spec.ts

```diff
@@ -0,0 +1,252 @@
+import { toNano } from "@ton/core";
+import { Blockchain, SandboxContract, TreasuryContract } from "@ton/sandbox";
+import { ComputePhaseErrorsTester as TestContract } from "./contracts/output/compute-phase-errors_ComputePhaseErrorsTester";
+import "@ton/test-utils";
+
+describe("compute phase errors", () => {
+    let blockchain: Blockchain;
+    let treasure: SandboxContract<TreasuryContract>;
+    let contract: SandboxContract<TestContract>;
+
+    beforeEach(async () => {
+        blockchain = await Blockchain.create();
+        blockchain.verbosity.print = false;
+        treasure = await blockchain.treasury("treasure", {
+            resetBalanceIfZero: true,
+        });
+
+        contract = blockchain.openContract(await TestContract.fromInit());
+
+        const deployResult = await contract.send(
+            treasure.getSender(),
+            { value: toNano("10000") },
+            null,
+        );
+
+        expect(deployResult.transactions).toHaveTransaction({
+            from: treasure.address,
+            to: contract.address,
+            success: true,
+            deploy: true,
+        });
+    });
+
+    // 0: success
+    it("should test exit code 0", async () => {
+        const sendResult = await contract.send(
+            treasure.getSender(),
+            { value: toNano("10") },
+            "0",
+        );
+
+        expect(sendResult.transactions).toHaveTransaction({
+            from: treasure.address,
+            to: contract.address,
+            success: true,
+            exitCode: 0,
+        });
+    });
```

Comment:
Looks like this can be abstracted into a helper function, which can be iterated over a list of tested exit codes. This should significantly reduce the boilerplate

Reply:
Good catch, will do!